app/services/force/short_form_service.rb

```diff
@@ -171,5 +171,15 @@ def self.format_agent(agent)
       agent.slice('Id', 'FirstName', 'LastName').merge('Active' => status)
     end
     private_class_method :format_agent
+
+    def self.attachment_endpoint(application, file)
+      file_types = ['Homebuyer education certificate', 'Loan pre-approval']
+      if file_types.any? { |doc_type| doc_type == file.document_type }
+        "/shortForm/file/#{application['id']}"
```

Comment:
I think if there aren't any errors or issues, we can start just using file for all of them. 

Just double checked with larry and he says it should be OK. We just were going to start off with only eligibility files originally because we thought it would be a bigger change